triagebot.toml

```diff
@@ -630,7 +630,7 @@ style-team = [
 "/src/stage0.json" =                         ["bootstrap"]
 "/tests/ui" =                                ["compiler"]
 "/src/tools/cargo" =                         ["@ehuss", "@joshtriplett"]
-"/src/tools/compiletest" =                   ["bootstrap"]
+"/src/tools/compiletest" =                   ["bootstrap", "@wesleywiser", "@oli-obk"]
```

Comment:
I was asleep when this PR was put up, but I can also add myself to this rotation since I have reasonable familiarity with compiletest :)

Reply:
Go ahead, more is better!